spec/foundation_rails_helper/flash_helper_spec.rb

```diff
@@ -6,32 +6,80 @@
     end
   end
 
+  it "handles symbol keys" do
+    allow(self).to receive(:flash).and_return({ "success" => "Flash message" })
+    node = Capybara.string display_flash_messages
+    expect(node).to have_css("div.alert-box.success", :text => "Flash message")
+  end
+
+  it "handles string keys" do
+    allow(self).to receive(:flash).and_return({ "success" => "Flash message" })
+    node = Capybara.string display_flash_messages
+    expect(node).to have_css("div.alert-box.success", :text => "Flash message")
+  end
+
+  it "displays multiple flash messages" do
+    allow(self).to receive(:flash).and_return({ "success" => "Yay it worked", "error" => "But this other thing failed" })
+    node = Capybara.string display_flash_messages
+    expect(node).
+      to  have_css("div.alert-box.success", :text => "Yay it worked").
+      and have_css("div.alert-box.alert",   :text => "But this other thing failed")
+  end
+
   it "displays flash message with overridden key matching" do
-    allow(self).to receive(:flash).and_return({:notice => "Flash message"})
+    allow(self).to receive(:flash).and_return({ "notice" => "Flash message" })
     node = Capybara.string display_flash_messages({:notice => :alert})
-    expect(node).to have_css("div.alert-box.alert")
+    expect(node).to have_css("div.alert-box.alert", :text => "Flash message")
   end
 
   it "displays flash message with custom key matching" do
-    allow(self).to receive(:flash).and_return({:custom_type => "Flash message"})
+    allow(self).to receive(:flash).and_return({ "custom_type" => "Flash message" })
     node = Capybara.string display_flash_messages({:custom_type => :custom_class})
-    expect(node).to have_css("div.alert-box.custom_class")
+    expect(node).to have_css("div.alert-box.custom_class", :text => "Flash message")
   end
 
   it "displays flash message with standard class if key doesn't match" do
-    allow(self).to receive(:flash).and_return({:custom_type => "Flash message"})
+    allow(self).to receive(:flash).and_return({ "custom_type" => "Flash message" })
     node = Capybara.string display_flash_messages
-    expect(node).to have_css("div.alert-box.standard")
+    expect(node).to have_css("div.alert-box.standard", :text => "Flash message")
+  end
+
+  context "when the flash hash contains devise internal data" do
+    before do
+      FoundationRailsHelper.configure do |config|
+        config.ignored_flash_keys += [:timedout]
+      end
+    end
+
+    it "doesn't raise an error (e.g. NoMethodError)" do
+      allow(self).to receive(:flash).and_return({ "timedout" => true })
+      expect{ Capybara.string display_flash_messages }.not_to raise_error
+    end
+
+    it "doesn't display an alert for that data" do
+      allow(self).to receive(:flash).and_return({ "timedout" => true })
+      expect(display_flash_messages).to be_nil
+
+      # Ideally we'd create a node using Capybara.string, as in the other examples
```

Comment:
Expecting nil is fine with me.  Maybe you could have done:
```rspec
expect(node.to_s).to_not have_css("div.alert-box")
```
But, I'm not sure if an empty string responds to css either.